custom/icds_reports/ucr/data_sources/person_cases_v2.json

```diff
@@ -2395,6 +2395,16 @@
             "constant": null
           }
         }
+      },
+      {
+        "display_name": null,
+        "property_path": [],
+        "datatype": "string",
+        "is_primary_key": false,
+        "column_id": "phone_number",
+        "is_nullable": true,
+        "type": "raw",
+        "property_name": "phone_number"
```

Comment:
it looks like the property on the case is called `contact_phone_number` in the ticket, or did I read that wrong

Reply:
`contact_phone_number` is a property on AWC location and `phone_number` is a property on person case.